fachhochschule-vorarlberg-author-date.csl

```diff
@@ -1,33 +1,63 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <info>
-    <title>Fachhochschule Vorarlberg (German)</title>
-    <title-short>FHV</title-short>
-    <id>http://www.zotero.org/styles/fachhochschule-vorarlberg</id>
-    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg" rel="self"/>
-    <link href="http://www.fhv.at/bibliothek/schreibzentrum/unterlagen-zum-wissenschaftlichen-arbeiten" rel="documentation"/>
+    <title>Fachhochschule Vorarlberg (author-date, German)</title>
+    <title-short>FHV Autor-Jahr</title-short>
+    <id>http://www.zotero.org/styles/fachhochschule-vorarlberg-author-date</id>
+    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg-author-date" rel="self"/>
```

Comment:
@adam3smith, so you're okay with the ID/title change? I saw some (German) discussion at https://github.com/citation-style-language/styles/pull/1933 on this.

@damnation333, if we make this change, the old and new style ID should be included in https://github.com/citation-style-language/styles/blob/master/renamed-styles.json, so that smart programs (like Zotero) can automatically update users from the old to the new style.